grumphp.yml.dist

```diff
@@ -14,9 +19,15 @@ parameters:
 
   extra_tasks:
     phpparser:
-      ignore_patterns: "%tasks.phpcs.ignore_patterns%"
+      ignore_patterns:
```

Comment:
Are you sure we need `extra_tasks`? It is from `grumphp` parameter, also these `ignore_patterns` and `triggered_by` are already defined in code-review's parameters.